### ISSUE TITLE
Twisted Error using 'pipenv install twisted

### ISSUE THREAD

**Issue by winrg** [bonfirefan/city-scrapers-pitt#36]
Upon trying to run scrapy, I got an error. reading through it, I discovered it required twisted. I installed it using "sudo pipenv install twisted", I got [this](https://pastebin.com/BGsavvb2) terminal out. I am doing this on a chromebook using the native debian VM.

**Comment by MartinSeibert** [comment]
I ha the same problem in a ubuntu 16.04 crouton on a chromebook. This stackoverflow solution worked for me: https://stackoverflow.com/questions/31457009/error-installing-twisted-for-python

I had to run the following before installing twisted: sudo apt-get install python3-dev

**Comment by jeslevine** [comment]
Install .whl file from here and make sure the c3x version of python matches
https://www.lfd.uci.edu/~gohlke/pythonlibs/#twisted

**Comment by bonfirefan** [comment]
1. Install C++ Build Tools from here (only select build tools, ~3GB): https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2017

2. Download wheel files from [the link jess posted](https://www.lfd.uci.edu/~gohlke/pythonlibs/#twisted)  
Make sure the filename matches your version of python in the (27, 33, 37) etc.

3. Run `pip install [wheel.whl]` where you downloaded the wheel

**Comment by biniona** [comment]
I am having a twisted bug on OSX 10.14.6. After entering my forked directory and running `pipenv install --dev --ignore-pipfile --three` . I try to run `scrapy genspider chi_housing "Chicago Housing Authority" http://www.thecha.org` and get the error `ModuleNotFoundError: No module named 'twisted'. ` I tried to run 

`pip install twisted ` and
`pip3 install twisted ` outside of pipenv shell

`pipenv install twisted ` before going into pipenv. 

All seemed to be successful but the error is still happening. 

Any ideas what I should try?

**Comment by biniona** [comment]
Solved the problem for Mac OSX Mojave twisted error! I think the issue had to do with me having too many versions of python on my machine and a bunch of dependencies got jumbled up. I attempted to deinstall all my versions of python and then reinstall python 3.7 by following these two links and it worked.

- https://apple.stackexchange.com/questions/284824/remove-and-reinstall-python-on-mac-can-i-trust-these-old-references

-  https://stackoverflow.com/questions/3819449/how-to-uninstall-python-2-7-on-a-mac-os-x-10-6-4

Also reinstalled pipenv during the process (a licensing error came up). I am down to talk about this more if anyone else gets the same error. 

SUMMARY: It had pretty much nothing to do with twisted. Seemed more to be an issue with my having weird linking and dependency stuff and I just had to delete all my python versions and reinstall the one I needed for this project.
